spec/views/layouts/user_mailer.html.erb_spec.rb

```diff
@@ -19,6 +19,10 @@
     expect(rendered).to have_css("img[src*='.mail']")
   end
 
+  it 'includes alt text for app logo that reads Login.gov logo' do
+    expect(rendered).to have_css("img[alt='Login.gov logo']")
```

Comment:
While this probably passes, it'd be good to make this assertion locale-independent.

```suggestion
    expect(rendered).to have_css("img[alt='#{t('mailer.logo', app_name: APP_NAME)}']")
```